packages/muelu/src/Graph/MatrixTransformation/MueLu_CoalesceDropFactory_def.hpp

```diff
@@ -195,7 +195,7 @@ namespace MueLu {
       ////////////////////////////////////////////////////
       // Remove this bit once we are confident that cut-based dropping works.
 #ifdef HAVE_MUELU_DEBUG
-      int distanceLaplacianCutVerbose = 0;
+   #   int distanceLaplacianCutVerbose = 0;
```

Comment:
Does this line of code compile? It looks like a `#` symbol instead of `//`

Reply:
Probably doesn't compile.   Fixed.